project/Dependencies.scala

```diff
@@ -70,6 +70,7 @@ object Dependencies {
   val playJson = "com.typesafe.play" %% "play-json" % playJsonVersion
   val playJsonJoda = "com.typesafe.play" %% "play-json-joda" % playJsonVersion
   val playIteratees = "com.typesafe.play" %% "play-iteratees" % "2.6.1"
+  val atomRenderer = "com.gu" %% "atom-renderer" % "0.7.10"
```

Comment:
Does this compile for `scala 2.12`? This is now a requirements for all new scala dependencies

Reply:
Not yet, but can do.

Reply:
You will have to. Otherwise @QuarpT is going to be very unhappy 😬  
Let us know if you need help to do it

Reply:
Thanks. It's already [done](https://github.com/guardian/atom-renderer/releases/tag/v0.8.0), I'm waiting for Maven to sync 😄